setup.py

```diff
@@ -37,23 +37,25 @@
     license='GPLv3',
     packages=find_packages(exclude=['tests*',
                                     'hyperspyui.plugins.user_plugins']),
-    requires=['hyperspy (>= 1.1.1)',
+    requires=['hyperspy (>= 1.3.0)',
```

Comment:
Is 1.3.0 actually required? The only change to hyperspy fronting code I see is in ` hyperspyui/overrides` and that should be possible to put in a `try`/`except: pass` block.

Reply:
Would you mind reverting this to 1.1.1 (here and below)? If we are unsure, it is better to leave it up to the user.

Reply:
Done!